pss/crypto/utils.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Swarm Authors
+// This file is part of the Swarm library.
+//
+// The Swarm library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Swarm library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Swarm library. If not, see <http://www.gnu.org/licenses/>.
+package crypto
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"sync"
+
+	ethCrypto "github.com/ethereum/go-ethereum/crypto"
+)
+
+// Utils contains utility methods for tests. Generates and stores asymmetric keys
+type Utils interface {
+	GenerateKey() (*ecdsa.PrivateKey, error)
+	NewKeyPair() (string, error)
```

Comment:
Not sure. I guess in some tests we just don't need the public key or store the pub/priv key. 
In fact, with `NewKeyPair` we just then recover the private key from the store in all cases:
```go
keys, err := utils.NewKeyPair()
if err != nil {
    return nil, err
}
privkey, err := utils.GetPrivateKey(keys)
```
When using `GenerateKey` we are doing it in one go. So perhaps we can change all tests to use `GenerateKey`.

Reply:
And if we remove `NewKeyPair` we can also remove `GetPrivateKey`, and the utils interface completely (because we will just use `ethCrypto.GenerateKey()`